docs/current_docs/manuals/administrator/integrations/248292-openshift.mdx

```diff
@@ -0,0 +1,46 @@
+---
+slug: /manuals/administrator/248292/openshift
```

Comment:
@gerhard @kpenfound can you please review this page? I have based it on https://archive.docs.dagger.io/0.9/488564/openshift-gitlab, modified to use a function call.